src/test-apps/happy/lib/WeaveDeviceManager.py

```diff
@@ -32,7 +32,19 @@
 import base64
 import json
 import set_test_path
-import WeaveDeviceMgr
+
+
+weaveDeviceMgrPath = os.environ['weaveDeviceMgrPath']
```

Comment:
Change the variable name to be ALL_CAPS_WITH_UNDERSCORES

Reply:
Done, thanks